### PR TITLE
DOC: describe workflow for spatially-offset traces 

### PR DESCRIPTION
**Pull request by bmorris3**
### Description

A workflow was added in #1556 to generate traces for spectral extractions based on existing traces. This PR adds to the narrative docs to explain the workflow.

[🐱](https://jira.stsci.edu/browse/JDAT-2724)

### Change log entry

- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: docs/specviz2d/plugins.rst [hunk 1]
```diff
@@ -78,11 +79,16 @@ Trace objects created outside of jdaviz can be loaded into the app via :py:meth:
...
-Once trace objects are loaded into the app, they can be offset (in the cross-dispersion direction)
-by selecting the trace label, entering an offset, and overwriting the existing data entry (or
-creating a new one) with the modified trace.
+Suppose you have 2D spectra of an extended source, and you have already created a trace that follows the
+bright central region of the source. It is possible to create a new trace, with the same 2D
+shape as the trace of the central region, but offset in the spatial direction. This might be useful
+for extracting spectra from the faint outer regions of the extended source, while using a trace computed from the
+brighter inner region. You can create a new trace based on the existing trace by clicking
+the "Trace" dropdown and selecting the existing trace. Then, offset it in the spatial direction by clicking or entering
+the spatial offset, and save it by creating a new trace or overwriting the existing trace entry.
```
Comment (pllim) [inline docs/specviz2d/plugins.rst:88]: Enter it where? Is it obvious without more explanation?

Comment (bmorris3) [reply docs/specviz2d/plugins.rst:88]: Yes I think it's clear, the offset should be entered into a box called "offset."

**Comment by bmorris3** [comment]
Thanks all!

File: docs/specviz2d/plugins.rst (hunk 1 above)
Comment (kecnry) [reply docs/specviz2d/plugins.rst:88]: my one thought/concern on this is that it is very tangential and specific to an "uncommon" workflow... but I don't have any ideas for where else to put this quite yet.

**Pull request merged**